revolv/settings.py

```diff
@@ -295,4 +291,5 @@
     ('en-us', 'English'),
 ]
 
-CHARGE_INSTRUMENT = False
+# be careful!
```

Comment:
I think we should expand this comment to talk about how the paypal keys work and how cards won't actually get charged unless there are non-sandbox keys. Also should put that somewhere more visible like the readme AND the docstring of PaymentInstrument